docs/source/index.md

```diff
@@ -16,91 +23,106 @@ support/index
 api-reference/index
 :::
 
-## Scylla Operator
-
-::::{grid} 1 1 1 2
-:reverse:
+```{hero-box}
```

Comment:
not sure about it, but shouldn't we also bump the version requirements in pyproject.toml?

Reply:
Are you referring to the theme:

    'sphinx-scylladb-theme (>=1.8.1,<2.0.0)',

This spec allows any version >=1.8.1 and <2.0.0, but `pyproject.toml` currently locks it to version 1.8.7, so shouldn't be necessary.